group_vars/all/public.yml

```diff
@@ -11,3 +11,6 @@ security_autoupdate_mail_to: '{{ OPS_EMAIL }}'
 
 # Certbot
 certbot_auto_renew_user: root
+
+# swapfile
+swapfile_size: false if (dedicated_server is undefined or not dedicated_server) else '512MB'
```

Comment:
Doesn't this need to go inside `"{{  }}"` to be processed by Jinja2?  I'd be surprised if it works like this.

Reply:
Yup, sorry, I think I forgot to push the latest amend.